extensions/wikia/EmbeddableDiscussions/EmbeddableDiscussionsController.class.php

```diff
@@ -31,6 +31,7 @@ public static function render( $input, array $args, Parser $parser, PPFrame $fra
 		$showLatest = filter_var( $args['latest'], FILTER_VALIDATE_BOOLEAN );
 		$itemCount = empty( $args['size'] ) ? self::ITEMS_DEFAULT : intval( $args['size'] );
 		$columns = empty( $args['columns'] ) ? self::COLUMNS_DEFAULT : intval( $args['columns'] );
+		$category = empty( $args['category'] ) ? null : ( new DiscussionsCategoryModel( $wgCityId ) )->getCategoryId( $args['category'] );
 
 		if ( $itemCount > self::ITEMS_MAX ) {
 			$itemCount = self::ITEMS_MAX;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Use "static" keyword instead of "self". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS2037)